.env-sample

```diff
@@ -1 +0,0 @@
-GRAPHQL_URL=
```

Comment:
why is this file deleted?

Reply:
Sorry misread initial configuration instructions, renamed files without doing a copy first

I added them back in a new commit